src/wallet/wallet.cpp

```diff
@@ -223,6 +224,7 @@ std::shared_ptr<CWallet> LoadWalletInternal(interfaces::Chain& chain, const std:
         return wallet;
     } catch (const std::runtime_error& e) {
         error = Untranslated(e.what());
+        status = DatabaseStatus::FAILED;
```

Comment:
In commit "bugfix: make LoadWallet assigns status always" (f32ddd81267cfca9f3c07006cef7c80f79a48527)

Would be good to use `DatabaseStatus::FAILED_LOAD` here too